sources/packages/api/src/database/entities/restriction_notification_type.ts

```diff
@@ -0,0 +1,17 @@
+/**
+ * Types of notification for restrictions.
+ */
+export enum RestrictionNotificationType {
```

Comment:
Please review the file name, it is supposed to be something like `restriction-notification-status.type.ts`